worlds/LauncherComponents.py

```diff
@@ -1,14 +1,15 @@
 from enum import Enum, auto
 from typing import Optional, Callable, List, Iterable
 
-from Utils import local_path, is_windows
+from Utils import local_path
 
 
 class Type(Enum):
     TOOL = auto()
-    FUNC = auto()  # not a real component
+    FUNC = auto()  # not a real component, do not use anymore
```

Comment:
might be better the other way around? i.e. `MISC = auto(); MISC = FUNC` ? anyway, this way we can't give a DeprecationWarning. My original idea would've been renaming `FUNC` -> `MISC`, and then adding a new `FUNC` which gets checked and replaecd in `Components.__init__` to generate a DeprecationWarning.